valhalla/midgard/util.h

```diff
@@ -420,17 +420,26 @@ template <class T> struct iterable_t {
   using iterator = T*;
```

Comment:
we ended up not using this (were going to use it for holding a range (c++20) in an existing container). it doesnt hurt but we didnt use it